maintainers/maintainer-list.nix

```diff
@@ -6427,6 +6427,10 @@
     githubId = 278013;
     name = "Tomasz Kontusz";
   };
+  kube-linter = {
+    name = "kube-linter Maintainers";
+    email = "rhacs-kube-linter-eng@redhat.com";
```

Comment:
This will mean that you won't get notifications from our bot.

I really doubt that anyone would ever bother sending an email.

Reply:
Thanks for pointing this out. Will have to discuss this again with the team and reply ASAP.